### ISSUE TITLE
shiny applications using reticulate cannot be interrupted / stopped

### ISSUE THREAD

**Issue by kevinushey** [rstudio/rstudio#3634]
cc: https://community.rstudio.com/t/rstudio-1-2-1030-crashing-with-shiny-and-reticulate/16004

---

To reproduce, try running this Shiny application:

```R
library(shiny)
library(reticulate)

ui <- fluidPage(
    textOutput("pythonout")
)

server <- function(input, output, session) {
    output$pythonout <- renderText({
        os <- import("os")
        os$getcwd()
    })
}

shinyApp(ui, server)
```

and then try stopping that Shiny application. Note that this does work fine in RStudio v1.1.

**Comment by dfalty** [comment]
Verified fixed on version 1.2.1090.

**Comment by Cjdebaere** [comment]
I'm using a newer version (1.2.1268) but when hosting a shiny app (using reticulate), I still cannot terminate the process. Have to restart R every time.
